libs/filaflat/include/filaflat/BlobDictionary.h

```diff
@@ -37,7 +37,7 @@ class BlobDictionary {
     }
 
     inline void addBlob(Blob&& blob) noexcept {
-        mBlobs.emplace_back(std::move(blob));
+        mBlobs.push_back(std::move(blob));
```

Comment:
Remind me why this preferable here?

Reply:
I don't think it's preferable, it's just that emplace_back() is for passing the parameters that the ctor would take. Here we're just emplacing_back with the copy-ctor, which doesn't do anything different than push_back.